docs/docsite/rst/playbook_guide/playbooks_filters.rst

```diff
@@ -118,6 +118,30 @@ You should note that, while this may seem like a useful filter for checking that
 
 .. _dict_filter:
```

Comment:
The label should stay next to the section it refers to. Now it is pointing to your new section "Transforming strings into lists".

Reply:
Fixed.